pnpm-workspaces/.aspect/cli/main.star

```diff
@@ -0,0 +1,36 @@
+# TODO: delete when all rules_js rules added to aspect-cli
+aspect.register_rule_kind("js_binary", {
+    "From": "@aspect_rules_js//js:defs.bzl",
+    "MergeableAttrs": ["data"],
+    "ResolveAttrs": ["entry_point", "data"],
+})
+
+def declare_main_js(ctx):
+    if len(ctx.sources) == 0:
+        ctx.targets.remove("main")
+        return
+
+    entry_point = ctx.sources[0].path.replace(".ts", ".js")
+
+    ctx.targets.add(
+        name = "main",
+        kind = "js_binary",
+        attrs = {
+            # The entry point file
+            "entry_point": entry_point,
+            "data": [
+                aspect.Label(
```

Comment:
This should really be:
```
aspect.Import(
   provider = "js",
   id = entry_point
)
```

...then orion should automatically determine where that entry came from and add the correct label. For some reason that isn't working here and I don't have time to look into it atm, but for this demo we know this label will always be correct.

Reply:
SG